print number from 1 to 10.

In [1]:
num = 1
while num <= 10:
    print(num)
    num += 1

1
2
3
4
5
6
7
8
9
10


print number from 10 to 1.

In [2]:
num = 10
while num >= 1:
    print(num)
    num -= 1

10
9
8
7
6
5
4
3
2
1


print the multiplication tableof a number N .

In [ ]:
num = int(input("enter number : "))
i = 1
while i <= 10:
    print(num * i)
    i += 1

updated version

In [ ]:
num = int(input("enter number"))
a = 1    # >>> koi bhi aap variable bana skte ho jaruri nahi hai ki (i) hi ho

while a <= 10:
    print(num , "*" , a , "=" , num * a)
    a += 1

55 * 1 = 55
55 * 2 = 110
55 * 3 = 165
55 * 4 = 220
55 * 5 = 275
55 * 6 = 330
55 * 7 = 385
55 * 8 = 440
55 * 9 = 495
55 * 10 = 550


print the element of thr following list using a loop.
[1, 4 , 9 , 16 , 25 , 36 , 49 , 64 , 81 , 100]

In [ ]:
num = [1, 4 , 9 , 16 , 25 , 36 , 49 , 64 , 81 , 100]
idx = 0

while idx < len(num):
    print(num[idx])
    idx += 1

# in sabka index print karna ho to.

print("index here of num")

num = [1, 4 , 9 , 16 , 25 , 36 , 49 , 64 , 81 , 100]
idx = 0

while idx < len(num):
    print(idx)
    idx += 1

1
4
9
16
25
36
49
64
81
100
index here of num
0
1
2
3
4
5
6
7
8
9


search of a number X i this tuple using loop.(1, 4 , 9 , 16 , 25 , 36 , 49 , 64 , 81 , 100)

In [ ]:
num = (1, 4 , 9 , 16 , 25 , 36 , 49 , 64 , 81 , 100)
idx = 0
x = 49
while idx < len(num):
    if (num[idx] == x ):
        print("found :" ,"idx = ", idx ,"," , "num =" , x) 
    idx += 1

found : idx =  6 , num = 49


simple version ya to idx ya number print krna ho to

In [ ]:
num = (1, 4 , 9 , 16 , 25 , 36 , 49 , 64 , 81 , 100)
idx = 0
x = 49

while idx < len(num):
    if (num[idx] == x):

        print(idx)
        print(x)

    idx += 1

6
49


Use a while loop to find the sum of numbers from 1 to 10.

In [ ]:
num = 1
sum = 0

while num <= 10:
    sum = sum + num
    num += 1
    
print(sum)

55


Digit count

In [1]:
num = int(input("Enter number : "))

count = 0

while num > 0:
    num = num // 10 # jab hum (double // ) ka use krte hai to divide hota hai or decimal point hat jata hai
    count += 1

print(count)

7


Use a while loop to reverse a given number.

In [ ]:
num = 789

reverse = 0

while num > 0:
    digit = num % 10
    reverse = reverse * 10 + digit
    num = num // 10

print("Reverse:", reverse)

Reverse: 987


Use a while loop to count the number of even digits in a given number.

In [ ]:
num = 789635874123

count = 0

while num > 0:
    digit = num % 10

    if digit % 2 == 0:
        count += 1

    num = num // 10

print("Even digits:", count)

Even digits: 5


Create a program that keeps asking the user to enter a password until the correct password is entered.

In [ ]:
password = 1234

while True:
    user_password = int(input("Enter password: "))

    if user_password == password:
        print("Access Granted")
        break
    else:
        print("Wrong Password")

Wrong Password
Wrong Password
Wrong Password
Wrong Password
Wrong Password
Wrong Password
Access Granted


print the name 10 time .

In [ ]:
name = 1

while name <= 10:
    print("suraj kumar")

    name += 1

suraj kumar
suraj kumar
suraj kumar
suraj kumar
suraj kumar
suraj kumar
suraj kumar
suraj kumar
suraj kumar
suraj kumar


1 to 10 tak print krna hai using while loop

In [ ]:
num = 1 
while num <= 10:
    print(num)
    num += 1

1
2
3
4
5
6
7
8
9
10


10 to 1 tak print krna hai using while loop

In [ ]:
num = 10

while num >= 1:
    print(num)
    num -= 1

10
9
8
7
6
5
4
3
2
1


write a program all even number between 1 to 50

In [ ]:
num = 1

while num <= 50:
    if (num % 2 == 0):
        print(num)
    num = num + 1

2
4
6
8
10
12
14
16
18
20
22
24
26
28
30
32
34
36
38
40
42
44
46
48
50


odd number 1 to 50 ( use of if)

In [ ]:
num = 1

while num <= 50:
    if (num % 2 != 0):
        print(num)
    num = num + 1

1
3
5
7
9
11
13
15
17
19
21
23
25
27
29
31
33
35
37
39
41
43
45
47
49


if n = 5 then output should  be 1 + 2 + 3 + 4 + 5 >> 15.

In [ ]:
n = int(input("enter user number : "))
sum = 0

while n >= 1:
    sum = sum + n
    n = n - 1
    
print(sum)

15


write a program to print this using while loop.  *, ** , *** , ****

In [ ]:
n = 1

while n <= 4:
    print("*" * n)
    n = n + 1


print("end of  while  loop")

*
**
***
****
end of  while  loop


write a program using whil e loop too print your name 5 times.

In [ ]:
n = 1

while n <= 5:
    print("suraj kumar")
    n = n + 1 

print("ENND OF WHILE LOOP OF YOUR NAME ")

suraj kumar
suraj kumar
suraj kumar
suraj kumar
suraj kumar
ENND OF WHILE LOOP OF YOUR NAME 


Print   a  table   (input a user).

In [ ]:
n = int(input("enter a number"))
i = 1
while i <= 10:
    print(n , " * " , i , " =" , n * i)
    i += 1

45  *  1  = 45
45  *  2  = 90
45  *  3  = 135
45  *  4  = 180
45  *  5  = 225
45  *  6  = 270
45  *  7  = 315
45  *  8  = 360
45  *  9  = 405
45  *  10  = 450


print number 1 to 10

In [ ]:
num = 1

while num <= 10:
    print(num)
    num += 1

1
2
3
4
5
6
7
8
9
10


1 to 10 even number

In [ ]:
num = 1

while num <=10:
    if num % 2 == 0:
        print(num)
    num += 1

2
4
6
8
10


1 to 10 odd number

In [ ]:
num = 1

while num <= 10:
    if(num % 2 != 0):
        print(num)
    num += 1

1
3
5
7
9


1 to 100 number ka sum

In [ ]:
num = 1
sum = 0

while num <= 100:
    sum = sum + num
    num += 1

print(sum)

5050


1 to 50 multiple of 5 print

In [ ]:
num = 1

while num <= 50:
    if num % 5 == 0:
        print(num)

    num += 1

5
10
15
20
25
30
35
40
45
50


input number lo or table print kro

In [3]:
num = int(input("enter a number"))
i = 1
while i <= 10:
    print(num , "*" , i , "=" , num  * i)              # >> 15 * 1 = 5

    i += 1

15 * 1 = 15
15 * 2 = 30
15 * 3 = 45
15 * 4 = 60
15 * 5 = 75
15 * 6 = 90
15 * 7 = 105
15 * 8 = 120
15 * 9 = 135
15 * 10 = 150


while loop se 10 se 1 tak numbers reverse order mein print karo.

In [ ]:
num = 10

while num >= 1:
    print(num)

    num -= 1

10
9
8
7
6
5
4
3
2
1


while loop se 1 se 10 tak sirf 3 ke multiples print karo.

In [ ]:
num = 1

while num <= 10:
    if (num % 3 == 0):
        print(num)

    num += 1

3
6
9


while loop se 1 se 50 tak un numbers ko print karo jo 4 se divisible hain.

In [ ]:
num = 1

while num <= 50:
    if ( num % 4 == 0):
        print(num)

    num += 1

4
8
12
16
20
24
28
32
36
40
44
48


while loop se kisi number ke digits count karo.

In [ ]:
num = int(input("enter number"))
i = 0 

while num > 0:
    num = num // 10
    i += 1

print(i)

9


digit count

In [ ]:
num = 12345678901234567812345678123456789123456789123456781234567890
count = 0 

while num > 0:
    num = num // 10

    count += 1

print("total digit : " ,count)

total digit :  62


digit count


In [ ]:
num = int(input("enter a number"))
suraj = 0

while num > 0:
    num = num // 10

    suraj += 1

print(suraj)

4


digit count

In [ ]:
num = int(input("enter a number"))
suraj_count = 0

while num > 0:
    num = num // 10
    suraj_count += 1

print("total_digit :" , suraj_count)

total_digit : 9


. while loop se 1 se 100 tak sirf 7 ke multiples print karo.

In [ ]:
num = 1

while num <= 100:
    if ( num % 7 == 0):
        print(num)

    num += 1

7
14
21
28
35
42
49
56
63
70
77
84
91
98


while loop se kisi number ke digits ka sum nikalo.

In [ ]:
num = 789654123 
sum = 0

while num > 0:
    last_digit = num % 10
    sum = sum + last_digit
    num = num // 10          # >> ishse decimal hat jata hai

    sum += 1

print("sum :" , sum)

sum : 54


while loop se kisi number ka factorial nikalo.
Example: 5 → 120

In [4]:
num = int(input("enter a number : "))
fact = 1

while num > 0:
    fact = fact * num

    num -= 1
    
print("factorial :" , fact)

factorial : 120


while loop se kisi number ko reverse karo.
Example: 1234 → 4321

In [ ]:
num  = 1234
reverse = 0

while num > 0:
    digit = num % 10
    reverse = reverse * 10 + digit

    num = num // 10

print(reverse)

4321


In [1]:
num = 1

while num <= 50:
    if num % 2 == 0:
        print(num)

    num += 1

2
4
6
8
10
12
14
16
18
20
22
24
26
28
30
32
34
36
38
40
42
44
46
48
50


In [3]:
num = 1

while num <= 50:
    if num % 2 != 0:
        print(num)

    num += 1

1
3
5
7
9
11
13
15
17
19
21
23
25
27
29
31
33
35
37
39
41
43
45
47
49
